pkg/quota/v1/evaluator/core/services.go

```diff
@@ -122,20 +124,40 @@ func (p *serviceEvaluator) Usage(item runtime.Object) (corev1.ResourceList, erro
 	result[corev1.ResourceServices] = *(resource.NewQuantity(1, resource.DecimalSI))
 	result[corev1.ResourceServicesLoadBalancers] = resource.Quantity{Format: resource.DecimalSI}
 	result[corev1.ResourceServicesNodePorts] = resource.Quantity{Format: resource.DecimalSI}
+	// check nodeport and load balancer only if the runtime svc is one of them.
```

Comment:
Drop this comment, since we're still setting both to zero normally. Also, go ahead and squash the commits so we don't remove and then re-add these lines in consecutive commits

Reply:
squashed.
@liggitt

Reply:
sorry for my careless。